### PR TITLE
Added PRelu to CoreML emitter.

### PR DESCRIPTION
**Pull request by galli-leo**
Note: I haven't tested yet if this gives the correct output, but I am assuming it should.

Also, you should consider adding .DS_Store to the .gitignore, since these files are very prevalant when working on a mac.

### CODE REVIEW

**Comment by kitstar** [comment]
Thanks @galli-leo !

**Comment by galli-leo** [comment]
Might have been a premature merge 😅 . Just started testing the conversion of insightface and outputs seem to differ by a lot from CoreML and MXNet version. Do you have an idea on the orders of parameters MXNet and CoreML have for activation layers? Or do you know an easy way to check that?

**Comment by kitstar** [comment]
Did you try if the MXNet->MXNet conversion works?

**Comment by galli-leo** [comment]
Ah that's a good idea, will try shortly.

**Comment by galli-leo** [comment]
Ok, so MXNet-> MXNet works (I had to add PRelu to the emitter though). MXNet -> CoreML still doesn't :/ Not sure if that's because of PRelu or something else. Last time I converted Torch -> CoreML (using a different framework), I had issues with the Batchnorm layers. Did you verify whether those pass the correct parameters?

**Comment by kitstar** [comment]
I haven't tested the CoreML emitter well (Formal test is on the way), just tested with VGG, inception networks from TF.

**Comment by galli-leo** [comment]
So I tried removing the PRelu layers entirely from both MXNet and CoreML models and the results are still way off :/ At least we know it's not solely the PRelu layer. I think the problem lies with the batchnorm layer, though I haven't found out what.

**Comment by galli-leo** [comment]
A small update: Doing MXNet -> Caffe (using this repo) then doing Caffe -> CoreML via coremltools, results in even more different results (all values seem to be in the million range for some reason).

**Comment by galli-leo** [comment]
Finally got it to work using: https://github.com/GarrickLin/MXNet2Caffe. So something is definitely off in the caffe and coreml emitters. Or maybe the error happens when reading the mxnet stuff (i.e. wrong conversion of variables) and then just works again since the emitter also makes some wrong assumptions?

**Comment by galli-leo** [comment]
I finally managed to get MXNet -> Caffe working with MMdnn. Had to do some manual editing in the generated file.  If it is desired, I can try and implement these changes in the caffe emitter? Mostly just adding support for PReLU and fixing Batchnorm with scaling.

MXNet -> CoreML is still not working, though I am suspecting this is due to the Batchnorm stuff. I asked the coremltools dev what exactly they are doing with the keras conversion and then I can try and fix that too.

**Comment by kitstar** [comment]
Many thanks! Please help us to improve CaffeEmit! And To CoreML part is under testing these days. We will ping you if coreml test and modification is finished. Then we can go on the MXNet -> CoreML part. 

Thanks again!
